HW06.Task2/Motorcycle.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+
+namespace HW06.Task2
+{
+    enum EngineType
+    {
+        Petrol,
+        Electro,
+        Hybrid,
+    }
+    class Motorcycle
+    {
+        public Guid Id { get; } = Guid.NewGuid();
+        public string Model { get; private set; }
+        public string Manufacturer { get; private set; }
+        public DateTime CreationDate { get; } = DateTime.Now;
+        public uint Odometre { get; private set; }
+        public Engine MotoEngine { get; private set; }
+
+        public Motorcycle(string model, string manufacturer, uint odometre, ushort volume, ushort power, string engineType) =>
+            (Model, Manufacturer, Odometre, MotoEngine) = (model, manufacturer, odometre, new Engine(volume, power, engineType));
+
+        public void Reset() => Console.WriteLine("Сброс в заводское состояние");
```

Comment:
![image](https://user-images.githubusercontent.com/3023032/124738559-b6e04d00-df21-11eb-8d20-faa8c9c395c1.png)
Should be private